src/components/Icons/icons/IconCheckBoxCheck.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I believe this component already exists - [IconCheck](https://github.com/zer0-os/zUI/blob/master/src/components/Icons/icons/IconCheck.tsx). 

In Figma, the icon name is `Check`, which means the zUI icon name will be `IconCheck`. We wrote a script to pull them all in so you shouldn't need to add any extras.

![Screen Shot 2022-12-03 at 2 58 59 PM](https://user-images.githubusercontent.com/12437916/205417125-e32141e6-ce8a-4ab1-bbdc-8390312192d5.png)

Reply:
Removed. 
As There is a slight difference in the design that's why I exported an SVG from the design and created a new ICON. As of now, I removed it.